workflow_session/pipeline.py

```diff
@@ -0,0 +1,34 @@
+import datajoint as dj
+
+from element_lab import lab
+from element_animal import subject, genotyping
+from element_session import session
+
+from element_animal.subject import Subject
+# from element_animal.genotyping import Sequence, BreedingPair, Cage,\
+#                                       SubjectCaging, GenotypeTest
+from element_lab.lab import Source, Lab, Protocol, User, Project, ProjectUser
+from element_lab.lab import ProjectKeywords, ProjectPublication, ProjectSourceCode
+from element_session.session import Session, SessionDirectory, SessionExperimenter
+from element_session.session import SessionNote, ProjectSession
+
+if 'custom' not in dj.config:
+    dj.config['custom'] = {}
+
+db_prefix = dj.config['custom'].get('database.prefix', '')
+
+__all__ = ['genotyping', 'session', 'Subject', 'Source', 'Lab', 'Protocol', 'User',
+           'Project', 'ProjectKeywords', 'ProjectPublication', 'ProjectSourceCode',
+           'ProjectUser', 'Session', 'SessionDirectory', 'SessionExperimenter',
+           'SessionNote', 'ProjectSession']
+
+# Activate "lab", "subject", "session" schema -------------
```

Comment:
```suggestion
# Activate "lab", "subject", "session", "genotyping" schemas -------------
```

Reply:
My thought process was that we shouldn't try to demonstrate use of the genotyping schema without example data. Should we activate and keep empty even without the didactic tools to help the user explore?